src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisCacheWrapper.cs

```diff
@@ -8,14 +8,16 @@ namespace OrchardCore.Redis.Services;
 /// <summary>
 /// Wrapper preventing the <see cref="RedisCache"/> to dispose a shared <see cref="IConnectionMultiplexer"/>.
 /// </summary>
-public sealed class RedisCacheWrapper : IDistributedCache, IDisposable
+/// <remarks>
+/// This class shouldn't implement <see cref="IDisposable"/> because it prevents the internal Redis cache service from being disposed of when
+/// a tenant is restarted, or the <see cref="IConnectionMultiplexer"/> that is shared across tenants would be disposed too.
+/// </remarks>
+public sealed class RedisCacheWrapper : IDistributedCache
```

Comment:
```suggestion
/// <remarks>
/// This is done by not disposing the `RedisCache` instance which would otherwise close the redis connection.
/// </remarks>
public sealed class RedisCacheWrapper : IDistributedCache
```